### ISSUE TITLE
pulumi/pagerduty v3.10.2 - pagerduty.Schedule & pagerduty.User : unknown time zone America/New_York

### ISSUE THREAD

**Issue by F-PLopes** [pulumi/pulumi-pagerduty#219]
### What happened?

We've updated the @pulumi/pagerduty version to latest 3.10.2 due to the EOL of ServiceEventRules and its replacement by EventOrchestrationService rule sets.

However, the upgrade is now blocked due to the error while creating Users and Schedules with the timeZone property.
If in the case of an User, that property is optional, it is required for Schedules.

### Expected Behavior

Users and Schedules to be created with the appropriate timeZone.

### Steps to reproduce

NPM versions used:

- "@pulumi/pagerduty": "3.10.2",
- "@pulumi/pulumi": "^3.74.0",

Code example:
```typescript
const schedule = new pagerduty.Schedule('SDLC Schedule',
      {
        name: 'SDLC On-Call Schedule',
        description: 'On call schedule',
        layers: [
          {
            name: "Engineering Team Office Hours",
            restrictions: this.getOfficeHours(),
            rotationTurnLengthSeconds: defaultRotationPeriod,
            rotationVirtualStart: "2021-02-01T08:00:00-00:00",
            start: "2021-02-01T08:00:00-00:00",
            users: teamMembers.map((usr) => usr.id),
          },
        ],
        teams: [team.id],
        timeZone: "America/New_York",
      },
      { deleteBeforeReplace: true }
    );

this.user = new pagerduty.User(
        user.name,
        {
          email: user.email,
          jobTitle: user.title,
          role: user.role,
          timeZone: "America/New_York",
          name: user.name,
        },
        { deleteBeforeReplace: true }
      );
```

Pulumi preview outputs:

    error: pagerduty:index/user:User resource 'User POC' has a problem: unknown time zone America/New_York. Examine values at 'User.TimeZone'.

     error: pagerduty:index/schedule:Schedule resource 'Team POC Stable SDLC Escalation Schedule' has a problem: unknown time zone America/New_York. Examine values at 'Schedule.TimeZone'.

### Output of `pulumi about`

CLI
Version      3.74.0
Go Version   go1.20.5
Go Compiler  gc

Plugins
NAME       VERSION
nodejs     unknown
pagerduty  3.10.2

Host
OS       Microsoft Windows 10 Pro
Version  10.0.19044 Build 19044
Arch     x86_64

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by jaxxstorm** [comment]
The upstream Terraform provider validates the timezones using `time.LoadLocation` 

https://github.com/PagerDuty/terraform-provider-pagerduty/blob/3b824ee4c23eae91849f8890fbc8931fc6959271/pagerduty/resource_pagerduty_schedule.go#L53

This method loads time location [from][https://pkg.go.dev/time#LoadLocation]:

LoadLocation looks for the IANA Time Zone database in the following locations in order:

    the directory or uncompressed zip file named by the ZONEINFO environment variable
    on a Unix system, the system standard installation location
    $GOROOT/lib/time/zoneinfo.zip
    the time/tzdata package, if it was imported


As I can see from your `pulumi about` output, you're using a Windows system, which means 3 of those options won't work.

There is an open issue for issues with `time.LoadLocation` in the Go repo here: https://github.com/golang/go/issues/38453

You will need to install the Go programming language runtime in order to use this package on Windows.

**Comment by t0yv0** [comment]
The fix is being released right now as v3.10.3.
